postgresql.go

```diff
@@ -442,34 +439,41 @@ func (h *postgreSQL) buildInsertSQL(q shared.Queryable, tableName string, column
 	return h.baseHelper.buildInsertSQL(q, tableName, columns, values)
 }
 
-func (h *postgreSQL) tableHasIdentityColumn(q shared.Queryable, tableName string) (bool, error) {
-	defer h.tablesHasIdentityColumnMutex.Unlock()
-	h.tablesHasIdentityColumnMutex.Lock()
+func (h *postgreSQL) tableHasIdentityColumn(tableName string) bool {
+	tableName = strings.Trim(tableName, `"`)
```

Comment:
I found this to be interesting, and it seems like a quirk. Is it intended the table name has quotes around it?

Reply:
I am not sure (i will check it later). As I remember we add quotes in a code somewhere at the begining of the process, so hand-written queries do not struggle with some crazy table names like names with spaces and so on

Reply:
I am worried about situation, when you have the same table name but in a different `table_schema`

Anyway your change does not change anything in that regard, so let's keep it